includes/section-menu-functions.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+function mainsite_section_menu_display( $selector, $content='' ) {
+    $content = trim( $content );
+    $list_items = $content ? trim( Section_Menus_Common::get_custom_menu_items( $content ) ) : '';
+    $auto_select = $list_items ? 'false' : 'true';
+
+    ob_start();
+?>
+    <nav class="sections-menu-wrapper" aria-label="Page section navigation" style="min-height: 60px;">
```

Comment:
style attribute

Reply:
We use a style attribute on our default layout for auto section menus. This value is manipulated by the javascript that automatically adds the menu items, so it's important for it to be set here instead of the css file.